src/Model/Photo.php

```diff
@@ -1182,7 +1182,7 @@ public static function uploadAvatar(int $uid, array $files, string $url = ''): s
 
 		$r = self::store($image, $uid, 0, $resource_id, $filename, $album, 4, self::USER_AVATAR);
 		if (!$r) {
-			logger::notice('profile image upload with scale 4 (300) failed');
+			logger::warning('profile image upload with scale 4 (300) failed');
```

Comment:
Same as above, is more context useful here?

Reply:
Done